vignettes/plots.Rmd

```diff
@@ -0,0 +1,83 @@
+---
+title: "Creating CMAP-themed plots"
```

Comment:
I'm wondering if we want to break up this vignette into multiple? It feels like covering helper functions, theme_cmap, and finalize_plot all in one vignette could be overwhelming. Or, is the idea here to develop the man files for each function thoroughly enough that a highly detailed "cookbooks" isn't necessary and this is more of a high level overview?

Reply:
I think finalize plot warrants its own detailed vignette. I think the current "plots" vignette can be turned into the "cookbook" we were envisioning, with sample code highlighting `theme_cmap` and each of the helper functions. I don't think it needs to get too technical about each one, since we spent time making sure the documentation was solid. We could add text like "See `?theme_cmap` for detailed documentation" to the cookbook to direct people there.